### ISSUE TITLE
Training

### ISSUE THREAD

**Issue by Sweetylrx** [hustvl/MapTR#69]
I trained the method on a single RTX 3090 card and could not achieve results in the paper. I have set args.autoscale_l = True, can you provide parameters for single GPU? The following results are after 14 epochs.
divider: 0.41260679562886554
ped_crossing: 0.3006982629497846
boundary: 0.4218177944421768
map: 0.3783742843402756

**Comment by LegendBC** [comment]
Sorry, we have not tested the single card setting. You can refer to https://github.com/facebookresearch/detr/issues/48#issuecomment-638689380 for scaling learning rate in your single-card setting.

**Comment by PlutoXN** [comment]
> 我在单张 RTX 3090 卡上训练了该方法，但无法达到论文中的结果。我已经设置了 args.autoscale_l = True，你能提供单 GPU 的参数吗？以下结果是 14 个 epoch 后的结果。 分隔线：0.41260679562886554 ped_crossing： 0.3006982629497846 边界：0.4218177944421768 地图：0.3783742843402756

Hello, how to set up single card training? Can you share the steps on how to modify it?
